goldstone/core/models.py

```diff
@@ -2634,3 +2633,38 @@ def integration(cls):
     def resourcetype(cls):
 
         return "routers"
+
+class EventDef(DailyIndexDocType):
+
+    """ Model of EventDef data structure, this defines how events get
+        generated from parsing queries and log indices """
+
+    event_uuid       = UUIDField(version=1, auto=True, primary_key=True)
+    event_name       = CharField(max_length=128)
+    event_desc       = CharField(max_length=1024)
+    event_protected  = True
```

Comment:
I don't have a very strong opinion either way, but was curious why you have prefixed all of the field names with event_.  I think they will get that automatically (actually event_def_) when the model is sync'd to the DB.